pkg/core/consensus/testing/consensus_test.go

```diff
@@ -26,7 +27,7 @@ func mockGenesis() (block.Block, block.Certificate) {
 // TestConsensus passing means the consensus phases are properly assembled
 func TestConsensus(t *stdtesting.T) {
```

Comment:
Is there a reason to disable the entire `TestConsensus` including the sandbox environment for testing consensus?

Reply:
Mostly because it didn't work with the design at one point in the refactor. I also figured, that maybe the test-harness served the same purpose, and this was created to enable quicker integration testing. If it adds extra testing value, I can add it back.

Reply:
I think it would be valuable to have a testbed for consensus that does not rely on the network and focuses solely on the consensus/chain. Corner cases and other scenarios might be simpler to be observed. Will create an issue about it